sktime/forecasting/model_selection/tests/test_tune.py

```diff
@@ -37,6 +38,12 @@
 TEST_METRICS = [MeanAbsolutePercentageError(symmetric=True), MeanSquaredError()]
 
 
+def _check_no_fitted_params_before_fitting(tuner):
```

Comment:
This is already tested by `test_raises_not_fitted_error`, duplicate, kindly remove